### PR TITLE
[REPL] Support loading cluster from Matter IDL

### PR DESCRIPTION
**Pull request by erjiaqing**
#### Problem
Python REPL contains generated code, which makes the user rebuild the REPL every time after they change anything in the code.

#### Change overview
- Adds a new template for generate all IDL with clusters & attribtues & commands
- Add python code to Parse IDL and refresh attribute / cluster / event cache
- Remove generated Objects.py
- "chip.clusters.Objects" is a class instead of a module, users should not observe any differences if they only use it as a namespace.

#### Testing
* As Objects.py is removed, the existing code in tests should be using the objects generated from IDL now.

### CODE REVIEW

File: src/controller/python/chip/clusters/idl.py [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+import enum
+import dataclasses
+import typing
+
+
+def CreateParser():
+    """
+    Generates a parser that will process a ".matter" file into a IDL
+    """
+    return Lark.open('../idl/matter_grammar.lark', rel_to=__file__, start='idl', parser='lalr', transformer=matter_idl_parser.MatterIdlTransformer())
```
Comment (andy31415) [inline src/controller/python/chip/clusters/idl.py:36]: this path does not seem to exist: ../idl is not a folder but rather something in scripts.

File: src/controller/python/templates/MatterIDL.zapt [hunk 2]
```diff
@@ -0,0 +1,125 @@
+// This IDL was generated automatically by ZAP.
```
Comment (andy31415) [inline src/controller/python/templates/MatterIDL.zapt:1]: This files looks duplicated. there should be no source code copy and paste for this.

Comment (erjiaqing) [reply src/controller/python/templates/MatterIDL.zapt:1]: No this file is not copy&pasted.

Unlike the MatterIDL.zapt under app/templates, this file does not rely on the .zap files (thus it won't emit endpoint config)

Comment (andy31415) [reply src/controller/python/templates/MatterIDL.zapt:1]: It still looks extreamly similar to MatterIDL.zapt, hence my copy&paste comment. Are you saying you wrote this from scratch by hand?

We should be sharing code as much as possible, if this involves a new template fine.

Could we just ignore the endpoint config for python and use the other template?

**Pull request closed without merging**